src/components/progress/progress.stories.ts

```diff
@@ -30,4 +30,4 @@ export const darkThemeRTL_TestOnly = (): string => html`
   ></calcite-progress>
 `;
 
-darkThemeRTL_TestOnly.parameters = { themes: themesDarkDefault };
+darkThemeRTL_TestOnly.parameters = { themes: themesDarkDefault, chromatic: { disableSnapshot: true } };
```

Comment:
Based on the above changes, is it intentional for this to have both the `_TestOnly` suffix and disabled snapshot?

Reply:
Only for now while we are running Screener and Chromatic side by side. I'll clean it up once we completely remove screener